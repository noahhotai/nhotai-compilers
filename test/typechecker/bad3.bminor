main: function integer () = {
    x: integer = 3;
    if ("string") {
        for (;;) {
            {
                return 3;
            }
            return 6;
        }
        return;
    }
}
