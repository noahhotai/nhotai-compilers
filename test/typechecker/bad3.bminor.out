Program parsed successfully.
Program resolved successfully.
if condition must be booleanProgram failed to typecheck.
